articles/native-platforms/wpf-winforms.md

```diff
@@ -39,22 +42,7 @@ There are three options to do the integration:
 To start with, we'd recommend using __Lock__. Here is a snippet of code to copy & paste on your project.
 Since we are using `await` (.NET 4.5 or greater), your method needs to be `async`:
 
-```cs
-using Auth0.Windows;
-
-var auth0 = new Auth0Client(
-	"@@account.namespace@@",
-	"@@account.clientId@@");
-
-var user = await auth0.LoginAsync(this);
-/*
-    Use this object to do wonderful things, e.g.:
-      - get user email => user.Profile["email"].ToString()
-      - get facebook/google/twitter/etc access token => user.Profile["identities"][0]["access_token"]
-      - get Windows Azure AD groups => user.Profile["groups"]
-      - etc.
-*/
-```
+@@snippet(meta.snippets.use)@@
```

Comment:
Let's separate this into 2 snippets, one on top of the other. The first 3 lines are the setup. Then it's the use. Makes sense? @ntotten